packages/salesforcedx-vscode-core/src/util/cliConfiguration.ts

```diff
@@ -30,3 +30,7 @@ export function showCLINotInstalledMessage() {
   );
   window.showWarningMessage(showMessage);
 }
+
+export function isSFDXContainerMode(): boolean {
+  return process.env.SFDX_CONTAINER_MODE ? true : false;
```

Comment:
There were previous discussions that checking for the string 'true' was too specific, so I refactored to only check for the presence of the env variable. This is also how other util methods check for env variables.